sos/plugins/__init__.py

```diff
@@ -1026,6 +1026,17 @@ def report(self):
         else:
             return html
 
+    def check_process_by_name(self, pr):
+        """Checks if a named process is listed in ps -ef output."""
```

Comment:
That's not actually what this method does: it returns `True` if the given `pr` string is a *substring* of any running process's `ps` "full format" listing.. i.e. command name, arguments, etc. That's probably **not** what is wanted for this interface.

I'm dubious that the `ps` command is even the best choice of tool here. We could walk the process tree ourselves (not that costly when just checking names), or use a more focused tool like `pgrep`.